openassessment/staff_grader/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from openassessment.staff_grader.api import locks_view
+
+urlpatterns = [
+    # Enhanced Staff Grader (ESG) locking
+    url(r'submission/(?P<submission_uuid>.+)/lock$', locks_view, name='openassessment-submission-lock')
```

Comment:
I gotta say I strongly oppose this name. We have a service named submissions that is already a bit too coupled (imo) to ORA and this would be very confusing. Also, should the course also be a part of the URL?

Reply:
LMS URL routing includes the course so the whole URL path is:
```
{lms}/courses/{course}/staff_grader/submission/{submission_id}/lock
```

I went with this naming because we're trying to lock a submission (well, really a staff workflow, but the more user-accessible piece is the submission) and it's under the `staff_grader` section to disambiguate from regular submission API(s)

Reply:
I still think it should be changed to avoid confusion. Would you be opposed to `workflow`?

Reply:
fwiw, from the staff_grader perspective, this is working on the submission... though I understand that terminology is muddy.  But having `workflow` tied to the submission id in this call would seem more odd to me